### PR TITLE
Better checks and logs in Hermes.

### PR DESCRIPTION
**Pull request by menivaitsi**

### CODE REVIEW

File: AppDB/backup/backup_recovery_helper.py [hunk 1]
```diff
@@ -371,7 +371,7 @@ def deploy_apps(app_paths):
     appscale_info.get_secret())
 
   # Wait for Cassandra to come up after a restore.
-  time.sleep(5)
+  time.sleep(15)
```
Comment (obino) [inline AppDB/backup/backup_recovery_helper.py]: This should have something similar to what we do during boot: check when the port cassandra listen to is open. Not quite sure in python, but open the TCP socket should be enough to ensure the daemon is listening.

File: AppDB/backup/gcs_helper.py [hunk 2]
```diff
@@ -112,7 +112,7 @@ def download_from_bucket(full_object_name, local_path):
     return False
 
   # Compare to GCS file size.
-  if not content['size'] < available_space:
+  if int(content['size']) >= int(available_space):
     logging.error('Not enough space on the VM to download a backup.')
```
Comment (obino) [inline AppDB/backup/gcs_helper.py]: The message shouldn' have 'VM' in it.

Comment (obino) [inline AppDB/backup/gcs_helper.py]: That is: remove 'on the VM'.

**Comment by menivaitsi** [comment]
https://ocd.appscale.com:8080/job/Daily%20Build/624/

**Pull request merged**